src/utils/FontManager.js

```diff
@@ -62,12 +65,16 @@ async function isColrFontSupported() {
         const wait = ms => new Promise((r, j)=>setTimeout(r, ms));
         await wait(500);
 
+        console.log("Drawing canvas to detect COLR support");
         context.drawImage(img, 0, 0);
         colrFontSupported = (context.getImageData(10, 10, 1, 1).data[0] === 200);
+        console.log("Canvas check revealed COLR is supported? " + colrFontSupported);
     } catch (e) {
-        console.error("Couldn't load colr font", e);
+        console.error("Couldn't load COLR font", e);
         colrFontSupported = false;
     }
+
+    console.log({colrFontSupported});
```

Comment:
This line seems redundant, as all the branches seem to have some other message as well.